web/react/components/setting_picture.jsx

```diff
@@ -41,10 +41,14 @@ module.exports = React.createClass({
                             {img}
                         </li>
                         <li className='setting-list-item'>
-                            {server_error}
-                            {client_error}
+                            {serverError}
+                            {clientError}
                             <span className='btn btn-sm btn-primary btn-file sel-btn'>Select<input ref='input' accept='.jpg,.png,.bmp' type='file' onChange={this.props.pictureChange}/></span>
+                            {!this.props.loadingPicture ?
```

Comment:
Agreed, didn't even think about it (or see the red apparently) thanks taking it out now.